File: Hello_Python/day07/myupdate.py
import pymysql

conn = pymysql.connect(host='localhost', user='root', password='python', db='python', charset='utf8', port=3305)


e_id = '5'
e_name = '6'
sex = '6'
addr = '6'

cursor = conn.cursor()
sql = f"""
    UPDATE emp 
    set e_name = '{e_name}', sex = '{sex}', addr = '{addr}'
    where e_id = '{e_id}' """

print(sql)
cnt = cursor.execute(sql)
print(cnt)

conn.commit()
cursor.close()
conn.close()

File: Hello_Node/myupdate.js
var mysql = require('mysql');
var connection = mysql.createConnection({
	host: 'localhost',
	user: 'root',
	password: 'python',
	database: 'python',
	port: 3305
});

connection.connect();

var e_id = '5';
var e_name = '6';
var sex = '6';
var addr = '6';

var sql = `
    update emp
    set 
        e_name='${e_name}',
        sex='${sex}',
        addr='${addr}'
    where 
        e_id='${e_id}'
`;

console.log(sql);

connection.query(sql, function (error, result) {
	console.log("-----------------------------------");
	console.log(error);
	console.log("-----------------------------------");
	console.log(result.affectedRows);

});

connection.end();
